[train-164][stage] cannot require amplitude-event.1.schema.json in container

@chenba I think maybe the path reference is wrong

```
jsonPayload: {
  EnvVersion: "2.0"   
  Fields: {
   error: "Error: Cannot find module './amplitude-event.1.schema.json'
Require stack:
- /fxa-shared/metrics/amplitude.js
- /app/server/lib/amplitude.js
- /app/server/lib/routes/post-metrics.js
- /app/server/lib/routes/index.js
- /app/server/lib/server.js
- /app/server/bin/fxa-payments-server.js"    
   stack: "
Require stack:
- /fxa-shared/metrics/amplitude.js
- /app/server/lib/amplitude.js
- /app/server/lib/routes/post-metrics.js
- /app/server/lib/routes/index.js
- /app/server/lib/server.js
- /app/server/bin/fxa-payments-server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/fxa-shared/metrics/amplitude.js:8:25)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)"    
  }
  Logger: "fxa-payments-server"   
  Pid: 1   
  Severity: 0   
  Timestamp: 1585931089247000000   
  Type: "uncaughtException" 
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1490)
┆Issue Number: FXA-1490

--- a/packages/fxa-payments-server/Dockerfile
+++ b/packages/fxa-payments-server/Dockerfile
@@ -39,7 +39,7 @@ RUN npm ci
 # Except, to avoid installing redis, we're just copying over the files
 # that are explicitly needed.
 COPY --chown=app:app ["fxa-shared/metrics/user-agent.js", "../fxa-shared/metrics/user-agent.js"]
-COPY --chown=app:app ["fxa-shared/metrics/amplitude-event.1.schema.json", "../fxa-shared/amplitude-event.1.schema.json"]
+COPY --chown=app:app ["fxa-shared/metrics/amplitude-event.1.schema.json", "../fxa-shared/metrics/amplitude-event.1.schema.json"]
 COPY --chown=app:app ["fxa-shared/metrics/amplitude.js", "../fxa-shared/metrics/amplitude.js"]
 COPY --chown=app:app ["fxa-shared/metrics/flow-performance.js", "../fxa-shared/metrics/flow-performance.js"]
 COPY --chown=app:app ["fxa-shared/express/index.js", "../fxa-shared/express/index.js"]

